syntax = "proto2"; //用于proto3.x编译时兼容proto2.x
package gym_rl;

message OfflineRlInfo
{
    optional bytes feature = 1;
    optional bytes next_feature = 2;
    optional double reward = 3;
    optional int32 done = 4;
    optional double value = 5;
    optional double neg_log_pis = 6;
    repeated int32 action_list = 7;
    optional double advantage = 8;
    optional double reward_sum = 9;
    optional bytes game_id = 10;
    optional bool is_train = 11;
    optional int32 is_game_over = 12;
    optional int32 action = 13;
    optional bytes uuid = 15;
    optional bytes next_Q_value = 16;
    optional double gamma_pow = 17;
    optional int32 frame_no = 18;
    optional bytes lstm_c = 19;
    optional bytes lstm_h = 20;
    optional int32 style = 21;
}

message RlFragmentInfo
{
    repeated OfflineRlInfo samples = 1;
}